app/views/claims/pages/start.en.html.erb

```diff
@@ -8,15 +8,25 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <p class="govuk-body">
-        Use this service to claim for mentors who supported, or intended to support, trainee teachers from September 2023 to July 2024.
+        Use this service to claim for mentors who supported, or intended to support, trainee teachers from September <%= academic_year.starts_on.year %> to July <%= academic_year.ends_on.year %>.
```

Comment:
The issue is that the dates here are non-standard compared to the academic year, which we decided runs from September to August. To include the months, it'd end up being:

```erb
<%= l(academic_year.starts_on, format: :month_and_year) %> to <%= t(academic_year.ends_on - 1 month, format: :month_and_year) %>
```

which seems _strange_.

Reply:
Yeah. I agree that hardcoding it to always "remove one month" from the end of the academic year would be janky.

What I'm trying to understand is the meaning & intention behind the content, and whether it'd possible to make it align with the Academic Year or Claim Window concepts that are now built into the app.

For example, maybe the content should actually say "August" rather than "July"? I don't know what the significance is of it being "July" here.

Reply:
I've forwarded the question to Product to see if we can change it.

Reply:
We've decided to drop the dynamic dates here, as the content likely needs to be revised.